docs/getting_started.md

```diff
@@ -1,63 +1,76 @@
 
-Jaeger client libraries are already open source and for now can be used
-with Zipkin backend. You can see a HOW-TO example in the blog post
-[Tracing HTTP request latency in Go with
-OpenTracing](https://medium.com/@YuriShkuro/tracing-http-request-latency-in-go-with-opentracing-7cc1282a100a).
+The simplest way to start the all in one docker image is to use the pre-built image published to DockerHub.
 
-##Running Jaeger in Docker Container
-
-Coming soon...
+```bash
+docker run -d -p5775:5775/udp -p16686:16686 jaegertracing/all-in-one:latest
+```
 
-##Tracing a Sample Application
-**Hot R.O.D. - Rides on Demand**
+You can then navigate to `http://localhost:16686` to access the Jaeger UI. 
 
-<https://github.com/uber/jaeger/tree/master/examples/hotrod>
+##Sample Application
+###HotROD (Rides on Demand)###
 
 This is a demo application that consists of several microservices and
 illustrates the use of the [OpenTracing API](http://opentracing.io).
 
 It can be run standalone, but requires Jaeger backend to view the
 traces.
 
-### Features
+####Features
 
 -   Discover architecture of the whole system via data-driven dependency
-    diagram
--   View request timeline & errors, understand how the app works
--   Find sources of latency, lack of concurrency
--   Highly contextualized logging
+    diagram.
+-   View request time line & errors, understand how the app works.
+-   Find sources of latency, lack of concurrency.
+-   Highly contextualized logging.
 -   Use baggage propagation to
 
-    -   Diagnose inter-request contention (queueing)
-    -   Attribute time spent in a service
+    -   Diagnose inter-request contention (queueing).
+    -   Attribute time spent in a service.
 
 -   Use open source libraries with OpenTracing integration to get
-    vendor-neutral instrumentation for free
+    vendor-neutral instrumentation for free.
 
-### Prerequisites
+####Prerequisites
 
 -   You need Go 1.7 or higher installed on your machine.
--   Requires a running Jaeger backend to view the traces.
-    -   See [Running Jaeger backend in docker](#running-jaeger-in-docker-container)
+-   Requires a [running Jaeger backend](#all-in-one-docker-image) to view the traces.
 
-### Installation
+####Running 
 
-```shell 
+```bash
 go get github.com/uber/jaeger
+cd $GOPATH/src/github.com/uber/jaeger
 make install_examples
+cd examples/hotrod
+go run ./main.go all
 ```
 
-### Running
+Then navigate to `http://localhost:8080`.
 
-```shell
-cd examples/hotrod
-go run ./main.go all
+##Client Libraries
+
+Look [here](client_libraries.md).
+
+##Running individual components
+Individual components can be run from source. They have their `main.go` in the `cmd` folder. 
+
+For e.g., to run the agent, you'll have to do the following
+
+```bash
+go get github.com/uber/jaeger
+cd $GOPATH/src/github.com/uber/jaeger
+make install
+cd agent
+go run ./main.go 
```

Comment:
this doesn't work. And don't need to `cd` iirc, just `go run ./cmd/agent/main.go`